src/compiler/utilitiesPublic.ts

```diff
@@ -906,7 +906,7 @@ export function getNonAssignedNameOfDeclaration(declaration: Declaration | Expre
         case SyntaxKind.JSDocEnumTag:
             return nameForNamelessJSDocTypedef(declaration as JSDocEnumTag);
         case SyntaxKind.ExportAssignment: {
-            const { expression } = declaration as ExportAssignment;
+            const expression = skipOuterExpressions((declaration as ExportAssignment).expression);

```

Comment:
Apparently this change increases the bind time somewhat, because we use `getNameOfDeclaration` in the binder. I think I'll revert it then, but first I'm gonna try to figure out why that adjustment isn't needed in the binder's use cases.

Reply:
There are so few changes in the benchmark that it could just be noise; I might try running it again (in faster mode) just to see.